feeds/ipq807x_v5.4/ipq807x/base-files/etc/hotplug.d/firmware/10-ath11k-caldata

```diff
@@ -16,9 +16,9 @@ ath11k_generate_macs() {
 	echo -ne \\x${mac3//:/\\x} >> /lib/firmware/ath11k-macs
 }
 
-ath11k_generate_macs_wf194() {
+ath11k_generate_macs_wf196() {
 	touch /lib/firmware/ath11k-macs
-	mac=$(grep BaseMacAddress= /dev/mtd14 | cut -dx -f2)
+	mac=$(grep BaseMacAddress= /dev/mtd18 | cut -dx -f2)
```

Comment:
Are the MTD partitions renumbered when running the 5.4 kernel?   On my WF-196 units running 4.4, `mtd14` seems to be the correct one.   I don't have access to a unit running 5.4 at the moment to confirm if they are renumbered though.

```
root@04b6be13a188:~# cat /proc/mtd | egrep "mtd14|mtd18"
mtd14: 00010000 00010000 "0:APPSBLENV"
mtd18: 00080000 00010000 "0:ETHPHYFW"
root@04b6be13a188:~# grep BaseMacAddress= /dev/mtd14 | cut -dx -f2
04B6BE13A188
root@04b6be13a188:~# grep BaseMacAddress= /dev/mtd18 | cut -dx -f2
root@04b6be13a188:~#
```

Reply:
I'm running mine with 5.4.  `mtd18` has the bootloader environment variable with `BaseMacAddress`.  Seems like the MTD partitions got renumbered in 5.4.